src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -228,13 +228,7 @@ public function filter(string $key, mixed $default = null, int $filter = \FILTER
             return $value;
         }
 
-        $method = debug_backtrace(\DEBUG_BACKTRACE_IGNORE_ARGS | \DEBUG_BACKTRACE_PROVIDE_OBJECT, 2)[1];
-        $method = ($method['object'] ?? null) === $this ? $method['function'] : 'filter';
-        $hint = 'filter' === $method ? 'pass' : 'use method "filter()" with';
-
-        trigger_deprecation('symfony/http-foundation', '6.3', 'Ignoring invalid values when using "%s::%s(\'%s\')" is deprecated and will throw an "%s" in 7.0; '.$hint.' flag "FILTER_NULL_ON_FAILURE" to keep ignoring them.', $this::class, $method, $key, \UnexpectedValueException::class);
```

Comment:
I like this helpful message, highlighting the flag `FILTER_NULL_ON_FAILURE` when using `filter`. But I'm not sure about the perf impact of keeping the `debug_backtrace` call.

Reply:
Yes, but I could put it back.

Reply:
well, for an exception, the stack trace will already tell you where this was called.